src/data/finset.lean

```diff
@@ -2551,6 +2551,44 @@ lemma disjoint_iff_disjoint_coe {α : Type*} {a b : finset α} [decidable_eq α]
   disjoint a b ↔ disjoint (↑a : set α) (↑b : set α) :=
 by { rw [finset.disjoint_left, set.disjoint_left], refl }
 
+/--
+Given a set A and a set B inside it, we can shrink A to any appropriate size, and keep B
+inside it.
+-/
+lemma exists_intermediate_set {A B : finset α} (i : ℕ)
+  (h₁ : card A ≥ i + card B) (h₂ : B ⊆ A) :
```

Comment:
It's quite unfortunate, but currently the simplifier is not on speaking terms with `≥`... we've adopted the silly workaround of only using `≤`. Hopefully such pains will be gone in a nearby future.